arm-network/2017-06-01/swagger/serviceCommunity.json

```diff
@@ -88,11 +88,11 @@
           },
           "description": "The prefixes that the bgp community contains."
         },
-		"isAuthorizedToUse": {
+        "isAuthorizedToUse": {
```

Comment:
@akshaysngupta I see your point but as @veronicagg mentioned, if you know which one is correct example / service or if you know who is the owner of the service, let's make sure that we understand which one is correct and let's fix that while we have an opportunity here :). Thanks!